quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/ProxyPropertyMappers.java

```diff
@@ -21,6 +21,10 @@ public static PropertyMapper<?>[] getProxyPropertyMappers() {
                         .transformer((v, c) -> proxyEnabled(null, v, c))
                         .paramLabel("headers")
                         .build(),
+                fromOption(ProxyOptions.PROXY_PROTOCOL_ENABLED)
+                        .to("quarkus.http.proxy.use-proxy-protocol")
+                        .paramLabel("PROXY protocol")
```

Comment:
Sorry, just one last thing. For consistency, we should have here something like `true|false` like [here](https://github.com/keycloak/keycloak/blob/db4a555ebfac6cbad2c133a83508204194294b9d/quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/HealthPropertyMappers.java#L16) or [here](https://github.com/keycloak/keycloak/blob/db4a555ebfac6cbad2c133a83508204194294b9d/quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/MetricsPropertyMappers.java#L19), or no label at all.

Reply:
Corrected now.